python-checks/src/test/resources/checks/regex/notExistedGroupUsagesCheck.py

```diff
@@ -0,0 +1,64 @@
+import re
+
+def pattern_compile_group_by_name():
+    pattern = re.compile(r"(?P<a>.)")
+    #                      ^^^^^^^^> 1 {{There is no group named 'b' in the regular expression.}}
```

Comment:
I don't think it makes sense to duplicate the issue message on secondary location. Please have a look at the Java implementation.
Potential improvement: with the Java implementation, if there is no named group at all in the regular expression, I think we might not have secondary location at all, which might make it hard to understand the issue.
In this case, we could have a secondary location on the whole regex saying `No named groups defined in this regular expression` (or perhaps simply duplicating the initial message in this case).

Reply:
done